dronecan/sensors/rc/1141.RCLinkStats.uavcan

```diff
@@ -0,0 +1,38 @@
+#
+# This definition describes metadata and statistics for a RF control link.
+# It provides information regarding a link's signal quality, as well as transmitter
+# power, and antenna information. It accomodates single-radio, and dual-radio
+# link architectures. It provides information for both uplink (control unit to airborne
+# receiver), and downlink (airborne receiver to control unit) data.
+#
+
+uint8 STATUS_RSSI_VALID = 1 # RSSI field is valid
+
+# Uplink - received signal strength (RSSI) for antennas 1 and 2 respectively. Ranges from 
+# -255 to 0. (An integer value of 100 means RSSI is -100dBm)
```

Comment:
Isn't RSSI a manufacturer-specific arbitrary measure between 0 and 255 from weakest to strongest, and dBm is an absolute measure that is a negative value (low) to 0 (full power)?

If so the terminology here is mixed - RSSI can't be in dBm